config/default/capm3/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/cluster-api-provider-metal3:main
+      - image: 192.168.111.1:5000/localimages/cluster-api-provider-metal3:latest
```

Comment:
We can't change this and ipam image below to point to local images, please revert. You should be able to use something https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/Makefile#L454-L462 on the fly while running e2e tests but not set as is in upstream repo

Reply:
I think those files in `/config` were changed when I've run the test and I have should not commit those. Reverted.
Sorry for that.